.github/workflows/pull-osbuild-modules.yml

```diff
@@ -0,0 +1,49 @@
+name: Update developer guide
```

Comment:
I would merge this into the existing action, because it feels like everything apart from the `Pull modules` step is shared anyway.

Reply:
Thank you, I've added a commit that merges both into a single workflow.